module/Jobs/config/router.config.php

```diff
@@ -51,18 +51,16 @@
                 'completion' => array(
                     'type' => 'Segment',
                     'options' => array(
-                        'route' => '/organization/:organizationId',
+                        'route' => '/organization/:organizationId[/:callback]',
                         'defaults' => array(
                             'controller' => 'Jobs/ApiJobListByOrganization',
                             'action' => 'index',
-                            'defaults' => array(
-                                'defaults' => array(
-                                    'organizationId' => 0
-                                ),
-                                'constraints' => array(
-                                    'organizationId' => '[a-f0-9]+',
-                                ),
-                            ),
+                            'organizationId' => 0,
+                            'callback' =>false
+                        ),
+                        'constraints' => array(
+                            'organizationId' => '[a-f0-9]+',
+                            'callback' => '[0-9a-z]+',
```

Comment:
Should the constraint for the callback not something like
`[A-Za-z_$][0-9a-zA-Z_$]*`
this will not block reserved names

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Grammar_and_types#Variables
> A JavaScript identifier must start with a letter, underscore (_), or dollar sign ($); subsequent characters can also be digits (0-9). Because JavaScript is case sensitive, letters include the characters "A" through "Z" (uppercase) and the characters "a" through "z" (lowercase).

https://www.ecma-international.org/ecma-262/6.0/#sec-names-and-keywords
http://www.unicode.org/reports/tr31/tr31-29.html#Table_Lexical_Classes_for_Identifiers